conduit/fair/data/datasets/acs.py

```diff
@@ -180,6 +238,9 @@ def __init__(
             features, label, group = dataset.df_to_numpy(acs_data)
             non_ohe_indexes, feature_groups = None, None
 
+        # Make sure `group` is zero-indexed.
+        group -= group.min()
```

Comment:
Using the inverse unique indices would robustify this as we often desire the labels to be contiguous as well as 0-indexed; perhaps the contiguity is guaranteed at some prior point, however

Reply:
Hmm, that's not a bad idea, though then I should print out the mapping somewhere so that the experimenter can reconstruct the original values